src/homeWork1/task2/fullFileWriter/fullFileWriter.js

```diff
@@ -0,0 +1,25 @@
+import csv from 'csvtojson'
+import fs from 'fs'
+import writeText from "../utils/writeText/writeText";
+
+async function convertCSVToJson(data) {
+    let text;
+    try {
+        text = await csv().fromString(data)
+    } catch (e) {
+        console.error(e)
+    }
+    return text
+}
+
+export default function fullFileWriter(csvFilePath, txtFilePath) {
+    fs.readFile(csvFilePath, 'utf8', async (err, data) => {
+        if (err) {
+            console.error(err)
```

Comment:
it can be written in one line `return console.error(err)`

Reply:
fixed

Reply:
oh, I see, sorry

Reply:
oh, I see, sorry